package.json

```diff
@@ -38,6 +38,7 @@
     "lint:codeOnly": "eslint \"{bin}/**/!(__testfixtures__)/*.js\" \"{bin}/**.js\"",
     "lint": "eslint \"./bin/*.js\" \"./test/**/*.js\" \"packages/**/!(node_modules)/*.test.js\"",
     "precommit": "lint-staged",
+    "commitmsg": "commitlint -E GIT_PARAMS",
```

Comment:
Can we keep this simply as `commit`?

Reply:
This is the configuration I got from `commitlint`

https://marionebl.github.io/commitlint/#/guides-local-setup?id=install-husky